guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,6 +4,22 @@
 [id="katello-headline-features"]
 == Headline Features
 
+For the full list of changes, see the https://github.com/Katello/katello/blob/KATELLO-{KatelloVersion}/CHANGELOG.md[Changelog].
```

Comment:
Don't we have the changes included in the document now?

Reply:
We do..The script still creates a link to git repo changelog. Removed the link here.